### DISCUSSION TITLE
Best Stratum/Proxy to use?

### DISCUSSION THREAD

**Discussion by BenzMo** [dreams-money/merged-mining-pool#8] [General]
Hey dreams-money, thanks for this release it looks awesome! I'm working with a CTO and we're trying to figure out how best to communicate via Stratum/Proxy etc with the pool server (if I'm saying that right). Do you have any recommendations? A-lot of the opensource stratum proxies seem to be no longer maintained and may not support merge mining.

**Comment by dreams-money** [comment]
This is a stratum pool

**Reply by BenzMo** [reply]
Well now I feel silly (I think) :D Thanks for the fast response!

**Reply by dreams-money** [reply]
Anytime!
